src/components/dialogs/payment.vue

```diff
@@ -193,19 +189,27 @@ export default class PaymentDialog extends Mixins(
     }
   }
 
+  /** Called when user clicks "Back" button. */
+  goBack () {
+    // disable validation
+    this.$refs.staffPaymentComponent && this.$refs.staffPaymentComponent.setValidation(false)
```

Comment:
The component originally doesn't exist (until the tab is rendered).

This is a call into the component to set state.

Reply:
Is optionally chaining the reference an option here?
`this.$refs.staffPaymentComponent?...`

Reply:
Hmm, I would think so, but I'm getting this:
```
error: Expected an assignment or function call and instead saw an expression (no-unused-expressions) at src/components/dialogs/payment.vue:195:5:
  193 |   goBack () {
  194 |     // disable validation
> 195 |     this.$refs.staffPaymentComponent?.setValidation(false)
      |     ^
```

Reply:
I think that, technically, optional chaining works above, but the linter is complaining about it.

I think that, as in C, JS is fine with expressions that aren't assigned to anything, as above, but the "no-unused expression" is trying to save us from ourselves.

I suppose I could write `const f == this.blah?.func()` but that's not really any cleaner.

Another option is `if (this.blah) this.blah.func()`.

I think what I have is OK, but if you prefer something else, lmk.

Reply:
All good! Was just curious 👍